source/_less/position.less

```diff
@@ -0,0 +1,144 @@
+@import 'vars';
+
+@arrow-corner-spacing: 12px;
+@arrow-inner-offset: -6px;
+@arrow-outer-offset: -7px;
+@arrow-rotation: rotate(-45deg);
+@arrow-size: 8px;
+@container-background-color: @gray-0;
+@container-border-color: @gray-300;
+@container-shadow-color: fade(@gray-900, 16%);
+
+[position-arrow] {
+  background-color: @container-background-color;
+
+  &:before,
+  &:after {
+    border-color: transparent;
+    border-style: solid;
+    display: block;
+    height: 0;
+    position: absolute;
+    width: 0;
+  }
+
+  &:before {
+    border-color: @container-border-color;
+    border-width: @arrow-size;
+    content: " ";
+    transform: @arrow-rotation;
+    z-index: -1;
+  }
+
+  &:after {
+    border-color: @container-background-color;
+    border-width: @arrow-size;
+    content: " ";
+    transform: @arrow-rotation;
+  }
```

Comment:
it's possible that the "arrow" could overlap element content if padding ever changes